### PR TITLE
8.0 incubator

### PR DESCRIPTION
**Pull request by lasley**
Fixed missing relations and activated Medical EMR data. Validated that everything is working + tests are passing, so this will make a solid starting point for incubation

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash commit history to a few, because as is, they are not very relevant

**Comment by lmignon** [comment]
@lasley Thank you for the changes. As @pedrobaeza you should squash commit history. For your information an other refactoring /cleanup is on this way #22

File: medical_emr_data/__openerp__.py [hunk 1]
```diff
@@ -81,6 +82,6 @@
     'qweb': [
 
     ],
-    "active": False,
-    "installable": False,
+    "active": True,
```
Comment (dreispt) [inline medical_emr_data/__openerp__.py]: Remove "active"

File: medical/__openerp__.py [hunk 2]
```diff
@@ -42,7 +43,7 @@
         'base',
         'product'
     ],
-    'author': "Odoo Medical Team,Odoo Community Association (OCA)",
+    'author': "Odoo Medical Team, LasLabs, Odoo Community Association (OCA)",
     'category': 'Generic Modules/Others',
     'application': True,
     'description': """
```
Comment (dreispt) [inline medical/__openerp__.py]: Move description to a README file

File: medical_emr/__openerp__.py [hunk 3]
```diff
@@ -36,7 +37,7 @@
 
     'name': 'Medical EMR : Electronic Medical Record management for Medical',
     'version': '1.0',
-    'author': "Medical Team,Odoo Community Association (OCA)",
+    'author': "Medical Team, LasLabs, Odoo Community Association (OCA)",
```
Comment (dreispt) [inline medical_emr/__openerp__.py:40]: Move description to a README file

**Comment by lasley** [comment]
Hmmm well I can't figure out how to squash these commits in the same branch, so I'm going to close this request and create another. Will include the requested changes, as well.

**Comment by lmignon** [comment]
@lasley `git rebase -i origin/8.0-incubator` is your firend

**Comment by lasley** [comment]
Dang, I was so close too! TIL :smile:

**Comment by lasley** [comment]
Ahhhhh ok so GitHub doesn't retroactively update the pull request by removing the old commits! Sweet we should be good from here, and now rebase is part of my toolbelt. Much cleaner now :+1:

**Comment by yvaucher** [comment]
@lasley after having done a `git rebase` locally, you will have to `git push -f` I think you missed that operation as see a lot of "added missing relation" commits

**Comment by lasley** [comment]
@yvaucher - Me too, that's why I was confused. Solid advice with the `-f`, that seems to have done the trick - thank you. I did some stupid merge things before knowing this that created some hefty merge conflicts with a rebase in the current branch though. I'm just going to create a new request, sorry about that everyone

**Comment by dreispt** [comment]
We need an OCA crash course on Git. It can get quite confusing.

**Pull request closed without merging**